src/main/java/org/sonarlint/intellij/analysis/SonarLintTask.java

```diff
@@ -115,14 +115,14 @@ public void run(ProgressIndicator indicator) {
     } catch (CanceledException e1) {
       console.info("Analysis canceled");
       return;
-    } catch (RuntimeException e) {
+    } catch (Throwable e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1181)